api/creator/queries/useFetchRawCreators.ts

```diff
@@ -2,28 +2,31 @@ import { useMemo } from 'react'
 import { creatorKeys, CREATOR_QUERY_KEYS } from 'api/creator/creatorKeys'
 import { useToaster } from 'providers/ToastProvider'
 import { CreatorParams } from 'models/creator/creatorParams'
-import { Creator } from 'models/creator'
 import { useInfiniteQuery } from 'react-query'
 import http from 'api/http'
+import { RawCreator } from '@/models/creator/rawCreator'
+import { RawCreatorParams } from '@/models/creator/rawCreatorParams'
+import { useCreatorAuth } from '@/providers/CreatorAuthProvider'
 
-const { CREATOR, GET } = CREATOR_QUERY_KEYS
+const { CREATOR, GET_RAW } = CREATOR_QUERY_KEYS
 
-const fetchCreators = async (params: CreatorParams): Promise<Creator[]> => {
-	const response = await http.get<Creator[]>(`${CREATOR}/${GET}`, { params })
+const fetchCreators = async (params: CreatorParams): Promise<RawCreator[]> => {
+	const response = await http.get<RawCreator[]>(`${CREATOR}/${GET_RAW}`, { params })
```

Comment:
`fetchCreators` function should be renamed to `fetchRawCreators`. I think I did the same thing on `fetchRawComicIssues` and `fetchRawComics`. Please confirm and if I did it that way on other places, stick to the rule here as well.

It's prob just an oversight on your end